README.md

```diff
@@ -129,6 +129,35 @@ pip install nxontology[viz]
 The extra `viz` dependencies are required for the `nxontology.viz` module.
 This includes [pygraphviz](https://pygraphviz.github.io/), which requires a pre-existing [graphviz](https://graphviz.org/) installation.
 
+## Development
+
+Some helpful development commands:
+
+```shell
+# create a virtual environment for development
+python3 -m venv .venv
+
+# activate virtual environment
+source .venv/bin/activate
+
+# install package for development
+pip install --editable ".[dev,viz]"
+
+# Set up the git pre-commit hooks.
+# `git commit` will now trigger automatic checks including linting.
+pre-commit install
+
+# Run all pre-commit checks (CI will also run this).
+pre-commit run --all
+
+# run tests
+pytest
```

Comment:
I'd suggest using `tox` for the tests and virtual environment management, but this good either way

Reply:
Okay I'll play with tox and update the readme once I get the hang of it.

Reply:
I can send an example as a PR